vibe_code_apps_20/wattlab/studio/ep_viz.py

```
@@ -369,6 +369,18 @@ def publish_run_for_studio(
     # Pointer so Twin auto-selects the latest agent publish
     pointer = dest.parent / "CURRENT_RUN.txt"
     pointer.write_text(str(dest.resolve()), encoding="utf-8")
+    # Best-effort world-writable so host agents can archive without root.
+    try:
+        import os
+
+        os.chmod(dest, 0o777)
```

Ruff (0.15.21)

[error] `os.chmod` setting a permissive mask `0o777` on file or directory

(S103)